src/main/java/com/icosillion/podengine/models/ITunesItemInfo.java

```diff
@@ -124,7 +124,7 @@ public EpisodeType getEpisodeType() {
 
         Element episodeTypeElement = this.parent.element(QName.get("episodeType", this.iTunesNamespace));
         if (episodeTypeElement == null) {
-            return this.episodeType = EpisodeType.FULL;
+            return this.episodeType = EpisodeType.UNKNOWN;
```

Comment:
Did you check to see if this would break any existing tests? Might be worth writing a test for this, in any case.